<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sniffer Dog Analyst</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="whatsThis">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_config">
       <property name="whatsThis">
        <string extracomment="test"/>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Configure</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QTextEdit" name="textEdit_pivots"/>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="lineEdit_regex"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Url regex :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_update_PR">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pivots (one per lines) :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_run">
       <attribute name="title">
        <string>Run</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_nbScrappers">
          <property name="maximumSize">
           <size>
            <width>404</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QTextBrowser" name="textBrowser_logs"/>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_stop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_search">
          <property name="text">
           <string>Sniffer Dog Go Go Go !</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Number of Scrappers :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_pause">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_nbProxiesInUse">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_confirm">
       <attribute name="title">
        <string>Explore</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_searchResults">
          <property name="text">
           <string>Regex Search</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_resetResults">
          <property name="text">
           <string>Reset results in database</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTextBrowser" name="textBrowser_results"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_exportResults">
          <property name="text">
           <string>Export csv</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_searchResults"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_results">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Proxies</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit_proxies"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_updateProxies">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QTableView" name="tableView_proxies"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
